### ISSUE TITLE
Duplicated labels in multiple terms 

### ISSUE THREAD

**Issue by shawntanzk** [qudt/qudt-public-repo#731]
There are multiple terms in QUDT with the same rdfs:label 
I am not sure if you allow different schemas to have different rdfs:label - but most of them are that way. There are also some errors in labelling. Below is a table of all duplicated labels I got from a sparql query over the following models 
"http://qudt.org/2.1/schema/qudt"
"http://qudt.org/2.1/vocab/unit"
"http://qudt.org/2.1/vocab/sou"
"http://qudt.org/2.1/vocab/quantitykind"

Here are a few categories

1) Example of duplicated label in same model: 

| concept1  | concept2| label|
-- | -- | --
http://qudt.org/vocab/quantitykind/Basicity | http://qudt.org/vocab/quantitykind/Acidity | Acidity

2) Example of duplicated label in different model:

| concept1  | concept2| label|
-- | -- | --
http://qudt.org/vocab/quantitykind/Acidity | http://qudt.org/vocab/unit/PH | Acidity

3) Example of mistake in labelling: 

| concept1  | concept2| label|
-- | -- | --
http://qudt.org/vocab/unit/M3-PER-MOL-SEC | http://qudt.org/vocab/unit/CentiM3-PER-MOL-SEC | Cubic Centimeter per Mole Second

I assume the first should be cubic meter per mole second

4) Example of potentially duplicated concepts

| concept1  | concept2| label|
-- | -- | --
http://qudt.org/vocab/unit/DEATHS-PER-1000I-YR | http://qudt.org/vocab/unit/DEATHS-PER-KiloINDIV-YR | Deaths per 1000 individuals per year

<details><summary>Full table of duplicated labels</summary>

| concept1                                                                         | concept2                                                                         | label                                              |
|----------------------------------------------------------------------------------|----------------------------------------------------------------------------------|----------------------------------------------------|
| http://qudt.org/vocab/quantitykind/Acidity                                       | http://qudt.org/vocab/quantitykind/Basicity                                      | Acidity                                            |
| http://qudt.org/vocab/quantitykind/Acidity                                       | http://qudt.org/vocab/unit/PH                                                    | Acidity                                            |
| http://qudt.org/vocab/quantitykind/Basicity                                      | http://qudt.org/vocab/quantitykind/Acidity                                       | Acidity                                            |
| http://qudt.org/vocab/quantitykind/Basicity                                      | http://qudt.org/vocab/unit/PH                                                    | Acidity                                            |
| http://qudt.org/vocab/unit/PH                                                    | http://qudt.org/vocab/quantitykind/Acidity                                       | Acidity                                            |
| http://qudt.org/vocab/unit/PH                                                    | http://qudt.org/vocab/quantitykind/Basicity                                      | Acidity                                            |
| http://qudt.org/vocab/unit/CASES-PER-1000I-YR                                    | http://qudt.org/vocab/unit/CASES-PER-KiloINDIV-YR                                | Cases per 1000 individuals per year                |
| http://qudt.org/vocab/unit/CASES-PER-KiloINDIV-YR                                | http://qudt.org/vocab/unit/CASES-PER-1000I-YR                                    | Cases per 1000 individuals per year                |
| http://qudt.org/vocab/quantitykind/CombinedNonEvaporativeHeatTransferCoefficient | http://qudt.org/vocab/quantitykind/EvaporativeHeatTransferCoefficient            | Combined Non Evaporative Heat Transfer Coefficient |
| http://qudt.org/vocab/quantitykind/EvaporativeHeatTransferCoefficient            | http://qudt.org/vocab/quantitykind/CombinedNonEvaporativeHeatTransferCoefficient | Combined Non Evaporative Heat Transfer Coefficient |
| http://qudt.org/vocab/unit/CentiM3-PER-MOL-SEC                                   | http://qudt.org/vocab/unit/M3-PER-MOL-SEC                                        | Cubic Centimeter per Mole Second                   |
| http://qudt.org/vocab/unit/M3-PER-MOL-SEC                                        | http://qudt.org/vocab/unit/CentiM3-PER-MOL-SEC                                   | Cubic Centimeter per Mole Second                   |
| http://qudt.org/vocab/quantitykind/Curvature                                     | http://qudt.org/vocab/quantitykind/CurvatureFromRadius                           | Curvature                                          |
| http://qudt.org/vocab/quantitykind/CurvatureFromRadius                           | http://qudt.org/vocab/quantitykind/Curvature                                     | Curvature                                          |
| http://qudt.org/vocab/unit/DEATHS-PER-1000I-YR                                   | http://qudt.org/vocab/unit/DEATHS-PER-KiloINDIV-YR                               | Deaths per 1000 individuals per year               |
| http://qudt.org/vocab/unit/DEATHS-PER-KiloINDIV-YR                               | http://qudt.org/vocab/unit/DEATHS-PER-1000I-YR                                   | Deaths per 1000 individuals per year               |
| http://qudt.org/vocab/unit/DEATHS-PER-1000000I-YR                                | http://qudt.org/vocab/unit/DEATHS-PER-MegaINDIV-YR                               | Deaths per Million individuals per year            |
| http://qudt.org/vocab/unit/DEATHS-PER-MegaINDIV-YR                               | http://qudt.org/vocab/unit/DEATHS-PER-1000000I-YR                                | Deaths per Million individuals per year            |
| http://qudt.org/vocab/quantitykind/DiffusionCoefficient                          | http://qudt.org/vocab/quantitykind/NeutronDiffusionCoefficient                   | Diffusion Coefficient                              |
| http://qudt.org/vocab/quantitykind/NeutronDiffusionCoefficient                   | http://qudt.org/vocab/quantitykind/DiffusionCoefficient                          | Diffusion Coefficient                              |
| http://qudt.org/vocab/unit/E                                                     | http://qudt.org/vocab/unit/ElementaryCharge                                      | Elementary Charge                                  |
| http://qudt.org/vocab/unit/ElementaryCharge                                      | http://qudt.org/vocab/unit/E                                                     | Elementary Charge                                  |
| http://qudt.org/vocab/quantitykind/GeneralizedForce                              | http://qudt.org/vocab/quantitykind/GeneralizedMomentum                           | Generalized Force                                  |
| http://qudt.org/vocab/quantitykind/GeneralizedMomentum                           | http://qudt.org/vocab/quantitykind/GeneralizedForce                              | Generalized Force                                  |
| http://qudt.org/vocab/quantitykind/GrowingDegreeDay_Cereal                       | http://qudt.org/vocab/unit/DEG_C_GROWING_CEREAL-DAY                              | Growing Degree Days (Cereals)                      |
| http://qudt.org/vocab/unit/DEG_C_GROWING_CEREAL-DAY                              | http://qudt.org/vocab/quantitykind/GrowingDegreeDay_Cereal                       | Growing Degree Days (Cereals)                      |
| http://qudt.org/vocab/unit/KiloP                                                 | http://qudt.org/vocab/unit/KiloPOND                                              | Kilopond                                           |
| http://qudt.org/vocab/unit/KiloPOND                                              | http://qudt.org/vocab/unit/KiloP                                                 | Kilopond                                           |
| http://qudt.org/vocab/unit/MDOLLAR-PER-FLIGHT                                    | http://qudt.org/vocab/unit/MegaDOLLAR_US-PER-FLIGHT                              | Million US Dollars per Flight                      |
| http://qudt.org/vocab/unit/MegaDOLLAR_US-PER-FLIGHT                              | http://qudt.org/vocab/unit/MDOLLAR-PER-FLIGHT                                    | Million US Dollars per Flight                      |
| http://qudt.org/vocab/quantitykind/ElectromagneticPermeability                   | http://qudt.org/vocab/quantitykind/Permeability                                  | Permeability                                       |
| http://qudt.org/vocab/quantitykind/Permeability                                  | http://qudt.org/vocab/quantitykind/ElectromagneticPermeability                   | Permeability                                       |
| http://qudt.org/vocab/unit/PicoFARAD-PER-M                                       | http://qudt.org/vocab/unit/PicoGM-PER-L                                          | Picofarad Per Meter                                |
| http://qudt.org/vocab/unit/PicoGM-PER-L                                          | http://qudt.org/vocab/unit/PicoFARAD-PER-M                                       | Picofarad Per Meter                                |
| http://qudt.org/vocab/quantitykind/NuclearSpinQuantumNumber                      | http://qudt.org/vocab/quantitykind/SpinQuantumNumber                             | Spin Quantum Number                                |
| http://qudt.org/vocab/quantitykind/SpinQuantumNumber                             | http://qudt.org/vocab/quantitykind/NuclearSpinQuantumNumber                      | Spin Quantum Number                                |
| http://qudt.org/vocab/unit/TONNE-PER-M3                                          | http://qudt.org/vocab/unit/TON_Metric-PER-M3                                     | Tonne Per Cubic Meter                              |
| http://qudt.org/vocab/unit/TON_Metric-PER-M3                                     | http://qudt.org/vocab/unit/TONNE-PER-M3                                          | Tonne Per Cubic Meter                              |
| http://qudt.org/vocab/unit/TONNE-PER-DAY                                         | http://qudt.org/vocab/unit/TON_Metric-PER-DAY                                    | Tonne Per Day                                      |
| http://qudt.org/vocab/unit/TON_Metric-PER-DAY                                    | http://qudt.org/vocab/unit/TONNE-PER-DAY                                         | Tonne Per Day                                      |
| http://qudt.org/vocab/unit/TONNE-PER-HR                                          | http://qudt.org/vocab/unit/TON_Metric-PER-HR                                     | Tonne Per Hour                                     |
| http://qudt.org/vocab/unit/TON_Metric-PER-HR                                     | http://qudt.org/vocab/unit/TONNE-PER-HR                                          | Tonne Per Hour                                     |
| http://qudt.org/vocab/unit/TONNE-PER-MIN                                         | http://qudt.org/vocab/unit/TON_Metric-PER-MIN                                    | Tonne Per Minute                                   |
| http://qudt.org/vocab/unit/TON_Metric-PER-MIN                                    | http://qudt.org/vocab/unit/TONNE-PER-MIN                                         | Tonne Per Minute                                   |

</details>

**Comment by shawntanzk** [comment]
sorry just noticed that we have included deprecated terms, the following 4 are not duplicated as its due to deprecation 

index | sub | bool
-- | -- | --
1 | <http://qudt.org/vocab/unit/CASES-PER-1000I-YR> | "true"^^<http://www.w3.org/2001/XMLSchema#boolean>
2 | <http://qudt.org/vocab/unit/DEATHS-PER-1000I-YR> | "true"^^<http://www.w3.org/2001/XMLSchema#boolean>
3 | <http://qudt.org/vocab/unit/DEATHS-PER-1000000I-YR> | "true"^^<http://www.w3.org/2001/XMLSchema#boolean>
4 | <http://qudt.org/vocab/unit/MDOLLAR-PER-FLIGHT> | "true"^^<http://www.w3.org/2001/XMLSchema#boolean>

**Comment by shawntanzk** [comment]
Also noticed the following 5 pairs have qudt:exactMatch to each other, which might make sense if they have the same label - not 100% sure why they arent merged though? 

 index | sub | obj
-- | -- | --
1 | <http://qudt.org/vocab/quantitykind/ElectromagneticPermeability> | <http://qudt.org/vocab/quantitykind/Permeability>
2 | <http://qudt.org/vocab/quantitykind/Permeability> | <http://qudt.org/vocab/quantitykind/ElectromagneticPermeability>
3 | <http://qudt.org/vocab/unit/TONNE-PER-M3> | <http://qudt.org/vocab/unit/TON_Metric-PER-M3>
4 | <http://qudt.org/vocab/unit/TON_Metric-PER-M3> | <http://qudt.org/vocab/unit/TONNE-PER-M3>
5 | <http://qudt.org/vocab/unit/TONNE-PER-DAY> | <http://qudt.org/vocab/unit/TON_Metric-PER-DAY>
6 | <http://qudt.org/vocab/unit/TON_Metric-PER-DAY> | <http://qudt.org/vocab/unit/TONNE-PER-DAY>
7 | <http://qudt.org/vocab/unit/TONNE-PER-HR> | <http://qudt.org/vocab/unit/TON_Metric-PER-HR>
8 | <http://qudt.org/vocab/unit/TON_Metric-PER-HR> | <http://qudt.org/vocab/unit/TONNE-PER-HR>
9 | <http://qudt.org/vocab/unit/TONNE-PER-MIN> | <http://qudt.org/vocab/unit/TON_Metric-PER-MIN>
10 | <http://qudt.org/vocab/unit/TON_Metric-PER-MIN> | <http://qudt.org/vocab/unit/TONNE-PER-MIN>

**Comment by steveraysteveray** [comment]
Thanks for your thoughtful post. We do not forbid multiple URIs from having the same label. Some reasons are discussed below.

First, a few responses to your specific examples:
- Category 1: Agreed, Concept 1 should be labeled "Basicity"
- Category 2: Agreed, Concept2 should be labeled "pH"
- Category 3: Agreed, "Cubic Meter per Mole Second" for Concept 1
- Category 3: Yes, you caught the fact that some are deprecated. This was part of our effort to be consistent with URI names. The reason we did not simply delete the deprecated version is to minimize breaking applications that use it.

Your examples, including the qudt:exactMatch examples, bring up some policy issues. For the reason mentioned above of not wanting to affect working applications by deleting entries, our approach has been to not delete any existing units or quantity kinds, but rather mark them as either deprecated or qudt:exactMatch when appropriate. 

The reason for having multiple qudt:exactMatch entries is to support familiarity by users, such as TONNE vs. TON_Metric. Another example is when the common interpretation might be ambiguous, such as unit:OZ, which might mean "ounce of mass" or "ounce of volume" or "ounce of force". We selected the most common interpretation to be "ounce of mass", so we include unit:OZ with that meaning,  unit:OZ_F for the ounce of force, and unit:OZ_VOL for ounce of volume. 

(Thanks to your bringing up these issues, I recognize we should make all this consistent, as in:
- Have the common version (unit:OZ) with a qudt:exactMatch to unit:OZ_Mass
- Add the more precise version (unit:OZ_Mass) with a qudt:exactMatch to unit:OZ
- Keep the existing unit:OZ_F, and unit:OZ_VOL.
Also do the same with unit:LB and other ambiguous units)

The same reasoning applies to some of your other examples where, for example, quantitykind:Permeability is commonly used to mean quantitykind:ElectromagneticPermeability (as opposed to quantitykind:VaporPermeability). We want users to be able to find what they are looking for.
